src/app-modules/docker-compose

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -x
+
+docker_compose_cmd=docker-compose
+docker_cmd=docker
+
+app_load() {
+    local url="$1"
```

Comment:
Unused.

If the intention is to leave it for future reference, then add also `local application_name="$3"`. Otherwise remove :+1:

Reply:
added the `application_name`.